internal/database/database.go:59:10: undefined: OpenSQLiteDatabase

## Data
- **Shiori version**: v1.7.0 branch
- **Database Engine**: SQLite (the default).
- **Operating system**: FreeBSD 14
- **CLI/Web interface/Web Extension**: CLI

## Describe the bug / actual behavior

Trying to build Shiori from source results in:

    internal/database/database.go:59:10: undefined: OpenSQLiteDatabase

## Expected behavior
The build should succeed.

## To Reproduce
Steps to reproduce the behavior:
1. `git clone https://github.com/go-shiori/shiori.git`
2. `git checkout v1.7.0`
3. `go build`
4. See error

## Notes
Likely not a bug; rather, I'm missing something in the build process.

--- a/internal/database/sqlite_noncgo.go
+++ b/internal/database/sqlite_noncgo.go
@@ -1,5 +1,5 @@
-//go:build linux || windows || darwin
-// +build linux windows darwin
+//go:build linux || windows || darwin || freebsd
+// +build linux windows darwin freebsd
 
 package database
 

